src/cloud/lib/i18n/fr.ts

```diff
@@ -508,6 +511,16 @@ const frTranslation: TranslationSource = {
 
   [lngKeys.GeneralYes]: 'Oui',
   [lngKeys.GeneralNo]: 'Non',
+
+  [lngKeys.ModalUnlockDashboardsTitle]: 'Unlock Dashboards',
+  [lngKeys.ModalUnlockDashboardsDescription]:
+    'Create unlimited dashboards to organize your docs, tasks.. in order to guide your workflow in your own way! Display the information the way you want to see it.',
```

Comment:
`your docs, tasks.. in order to ...` -> `your docs, tasks, etc., in order to ...`